src/common/commonutils/UserUtils.c

```diff
@@ -828,28 +834,28 @@ int RemoveUser(SimplifiedUser* user, bool removeHome, OsConfigLogHandle log)
     }
     else if (0 == user->userId)
     {
-        OsConfigLogInfo(log, "RemoveUser: cannot remove user with uid 0 ('%s' %u, %u)", user->username, user->userId, user->groupId);
+        OsConfigLogInfo(log, "RemoveUser: cannot remove user with uid 0 (%u, %u)", user->userId, user->groupId);
         return EPERM;
     }
 
     if (NULL != (command = FormatAllocateString(commandTemplate, removeHome ? "-f -r" : "-f", user->username)))
     {
         if (0 == (status = ExecuteCommand(NULL, command, false, false, 0, 0, NULL, NULL, log)))
         {
-            OsConfigLogInfo(log, "RemoveUser: removed user '%s' (%u, %u, '%s')", user->username, user->userId, user->groupId, user->home);
+            OsConfigLogInfo(log, "RemoveUser: removed user %u", user->userId);
 
             if (DirectoryExists(user->home))
             {
-                OsConfigLogInfo(log, "RemoveUser: home directory of user '%s' remains ('%s') and needs to be manually deleted", user->username, user->home);
+                OsConfigLogInfo(log, "RemoveUser: home directory of user %u remains and needs to be manually deleted", user->userId);
```

Comment:
userId instead of user

Reply:
No, this is an user account, or in short, user. Not an 'userId'. UIDs identify user accounts, do not replace them.